src/Client/Im/Main.purs

```diff
@@ -727,7 +727,8 @@ toggleConnectedWebSocket isConnected model@{ hasTriedToConnectYet, errorMessage
             else
                   [ pure $ Just UpdateDelivered ]
       where
-      lostConnectionMessage = "Connection to the server lost. Attempting to automatically reconnect..."
+      lostConnectionMessage = "Connection to the server lost. Attempting to automatically reconnect... \n\
```

Comment:
Have you checked this message on mobile? It might be too long

Reply:
Well it's 3 lines long in my browser when I switch to mobile view, but it works.